content/docs/start/data-access.md

```diff
@@ -4,10 +4,13 @@ title: 'Get Started: Data Access'
 
 # Get Started: Data Access
 
-Okay, now that we've learned how to _track_ data and models with DVC and how to
-version them with Git, next question is how can we _use_ these artifacts outside
-of the project? How do I download a model to deploy it? How do I download a
-specific version of a model? How do I reuse datasets across different projects?
+Okay, now that we've learned how to _track_ data files in DVC and how to version
+them with Git. _Models_ in a machine learning project are also files written and
```

Comment:
I like the change, but it feels a bit misplaced - it splits the previously written paragraph which was written as a single piece. It feels like the whole section will be about models after these two sentences, but then we go with artifacts, and go with `How do I reuse datasets across different projects?`.

Also, I'm not sure it addressed the issue https://github.com/iterative/dvc.org/issues/2096 - it feel that we should put your sentence somewhere earlier? wdyt?

I'm even thinking if it worth creating a separate section Models Versioning ? To be extra explicit, to optimize SEO. Otherwise it'll be a constant battle that people read Data Versioning and do not connect dots (expected).

Reply:
Btw, section itself can be super simple first. That explains that models are the same as data for us pretty much.

Reply:
Another alternative is to change https://dvc.org/doc/start/data-versioning instead to avoid the confusion in the first place. I'm thinking specifically around "few commands that you can run along with git to track large files, directories, or ML models".

Reply:
>  I'm thinking specifically around "few commands that you can run along with git to track large files, directories, or ML models".

Unfortunately, I think it won't be enough. It's hard to beat the "Data Versioning" title.

Reply:
I agree that another document about model versioning between *Data Versioning* and *Data Access* might be better. It can be an overview of changing runtime parameters for models through `params.yaml` or run-cache. These could be a bit complex for a *Get Started* document but a simple *we put these parameters here and run this script and create these different models* kind of introduction can be written I think.

But usually people use already existing models at the beginning. We may need to discuss *access* first and *versioning* later.

Maybe we can just remove models from these documents and focus on data. *Data Versioning ➡️  Data Access ➡️ Model Versioning* can be another path. 

We can also discuss *Model Versioning* as a section at the end of Data Versioning for now. 

I'm fine with any of these solutions. We can continue to discuss this in #2096

BTW mentioning just the issue number (without "fix", "fixes") in the title should be OK for automated closing of issues. (At least that's what VS Code did while sending the PR 😃 ) I'll put "fixes" for clarity though.

Reply:
While reading #2195 I thought a way without much overhaul is adding _Model_ to _Data Versioning_ title and make it _Data and Model Versioning_ and describe that model versioning in a section. What I had in mind is already described in later sections of Getting Started.

Reply:
My take on this issue is that the term "model" is sprinkled in for SEO purposes but we weren't actually treating models different to a data file. That seemed to cause confusion for a user so we're trying to clarify the language and the easiest path is my suggestion + the page title change for now.

If we want to re-do the GS to have more dedicated model versioning material we should probably open an issue for that and wait until after #2195 which has some overlap (repurposes the **Experiments** page for ML-specific tools, and could also focus on model versioning).

Reply:
Sounds good to me, @iesahin @jorgeorpinel !

Reply:
should we also rename Data Access?

Reply:
"Access Data and Models"? Yes perhaps. Or "Access Artifacts" perhaps

Reply:
Artifacts - we'll have to define what artifact is then, unfortunately. Feels like we should not be overoptimizing this and do the simplest, the most explicit thing.

Reply:
This should be done in 8dea9639424e3928ec83bb56716ae2dee022d873 (in the new PR #2214 )

Thanks.